generator.yaml

```diff
@@ -1,8 +1,7 @@
 ignore:
```

Comment:
Ditto my q above; can we just remove this `ignore:` section now?

Reply:
Yup, removed.